spk/python312/crossenv/requirements-rpds_py.txt

```diff
@@ -0,0 +1,29 @@
+# with default crossenv the build of rpds_py>=0.23.0 has issue with maturin
+# with this minimal crossenv it succeeds.
+# 
+# related topics:
+# see https://github.com/crate-py/rpds/issues/120 
+# and https://mail-index.netbsd.org/tech-pkg/2025/03/20/msg030703.html
+# 
+# https://github.com/crate-py/rpds pyproject.toml declares the license metadata regarding pep639.
+# When building with default crossenv, it uses maturin that fails with:
+#  💥 maturin failed
+#    Caused by: pyproject.toml at /tmp/pip-wheel-1y_xcuh3/rpds-py_a98e895e9a524b8c92737c8eff981a64/pyproject.toml is invalid
+#    Caused by: TOML parse error at line 11, column 17
+#     |
+#  11 | license-files = ["LICENSE"]
+#     |                 ^^^^^^^^^^^
+#  wanted string or table
+#  
+#  Error running maturin: Command '['maturin', 'pep517', 'write-dist-info', '--metadata-directory', '/tmp/pip-modern-metadata-2vjpzv9m', '--interpreter', '/spksrc/spk/flexget/work-aarch64-6.2.4/crossenv-default/cross/bin/python']' returned non-zero exit status 1.
+#
```

Comment:
@th0ma7 any idea why we have this issue with default crossenv?
should I do further analyses?

Reply:
Couldn't this be related to the no isolation option added recently?

Reply:
Just _isolated_ the issue.

project.toml of rpds_py declares:
```
[build-system]
requires = ["maturin>=1.2,<2.0"]
build-backend = "maturin"
```

**but it requires `maturin>=1.8.2` (found by analysis).**

It was not the reduced crossenv, but the use of `maturin==1.8.6` vs. `maturin==1.8.1` that fixed the build of rpds_py.

Is it ok to update the default crossenv with latest maturin `maturin==1.8.6` ?

_it took some time to find this, because I always had to build flexget after `make clean`, after changes of crossenv definition files; when built after `make crossenvclean` I got random results_

Reply:
Others also had problems:
 https://mail-index.netbsd.org/tech-pkg/2025/03/20/msg030718.html

Reply:
issue reported upstream:
https://github.com/crate-py/rpds/issues/136